src/System.Reflection.Metadata/src/System/Reflection/Metadata/MetadataReader.cs

```diff
@@ -96,7 +95,7 @@ public unsafe MetadataReader(byte* metadata, int length, MetadataReaderOptions o
             this.Block = new MemoryBlock(metadata, length);
 
             _options = options;
-            this.Utf8Decoder = utf8Decoder;
+            this.StringDecoder = utf8Decoder;
```

Comment:
Can't rename the parameter - it's already public API. I renamed the property to be consistent with ```StringComparer``` .